src/core/shape/plane.js

```diff
@@ -26,21 +26,22 @@ class Plane {
      *
      * @param {Vec3} [normal] - Normal of the plane. The constructor copies this parameter. Defaults
      * to {@link pc.Vec3#UP}.
-     * @param {Vec3} [point] - Point position on the plane. The constructor copies this parameter.
-     * Defaults to {@link pc.Vec3#ZERO}.
+     * @param {Vec3} [distance] - The distance from the plane to the origin, along its normal.
+     * Defaults to 0.
      */
-    constructor(normal = Vec3.UP, point = Vec3.ZERO) {
-        this.set(normal, point);
+    constructor(normal = Vec3.UP, distance = 0) {
+        this.normal.copy(normal);
+        this.distance = distance;
```

Comment:
Maybe declare this above the constructor too?

Reply:
the member is there, but not initialized, to avoid initializing it twice - is that what you meant?